integrationtests/gitjob/controller/controller_test.go

```diff
@@ -345,6 +367,30 @@ var _ = Describe("GitJob controller", func() {
 
 				return string(job.UID) != string(newJob.UID)
 			}).Should(BeTrue())
+			// it should log 2 events
+			// first one is to log the new commit from the poller
+			// second one is to inform that the job was created
```

Comment:
```suggestion
			// it should log 3 events
			// first one is to log the new commit from the poller
			// second one is to inform that the job was created
			// third one reports on the job being deleted because of ForceUpdateGeneration
```